test/DynamoCoreTests/Nodes/DropDownTests.cs

```diff
@@ -98,6 +113,31 @@ public void OpenXmlDYNwithSelectionNode()
             Assert.AreEqual("1", node.GetType().GetProperty("SelectedString").GetValue(node));
         }
 
+        [Test]
+        public void GetSelectedStringFromItemShouldReturnString()
+        {
+            // Initialize the node
+            var node = new myDropDown();
+            node.PopulateItems();
+            //node.GetSelectedStringFromItem(node.Items.FirstOrDefault());
+            Assert.AreEqual("Monday", node.GetType().GetMethod("GetSelectedStringFromItem").Invoke(node, new object[] { node.Items.FirstOrDefault() }));
```

Comment:
Even after I exposed `GetSelectedStringFromItem`, I still could not invoke it. I checked the test setup, it should be fine. So this remains a mystery for me.. Now my machine refuse to run the test. Going to try my home machine.